File: Database/Models/PhysicalStateRecord.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FitnessApp.Database.Models
{
    public partial class PhysicalStateRecord: ID
    {
        [Required]
        public DateTime? Date { get; set; }
        [Required]
        public double? Weight { get; set; }
        [Required]
        public double? Height { get; set; }

        [ForeignKey(nameof(UserId))]
        public virtual User User { get; set; } = null!;
        public int UserId { get; set; }
    }
}

File: ClientApp/src/models/PhysicalStateRecord.js
export default class PhysicalStateRecord {
    date = new Date();
    weight = 0;
    height = 0;
    user = null;
    userId = 0;

    constructor(date,weight,user,userId) {
        this.date = date;
        this.weight = weight;
        this.user = user
        this.userId = userId;
    }
}